### PR TITLE
✅ Improve internal validators test coverage

### PR DESCRIPTION
**Pull request by lig**
## Change Summary

`pydantic/_internal/_validators.py` is covered as much is possible with exception of #1569 

## Related issue number

None so far

## Checklist

* [x] Unit tests for the changes exist
* [x] Tests pass on CI and coverage remains at 100%
* [x] Documentation reflects the changes where applicable
* [ ] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)
* [x] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

Selected Reviewer: @samuelcolvin

### CODE REVIEW

**Comment by lig** [comment]
please review

File: Makefile [hunk 1]
```diff
@@ -56,6 +56,8 @@ test: .pdm
 testcov: test
 	@echo "building coverage html"
 	@pdm run coverage html
+	@echo "building coverage lcov"
+	@pdm run coverage lcov
```
Comment (lig) [inline Makefile:60]: This one is a suggestion. I would like to add `coverage.lcov` generation. There are tools for IDEs that usually read this format. I use this one https://marketplace.visualstudio.com/items?itemName=markis.code-coverage

Comment (samuelcolvin) [reply Makefile:60]: 👍 fine by me

File: pydantic/_internal/_generate_schema.py [hunk 2]
```diff
@@ -991,7 +991,7 @@ def _pattern_schema(self, pattern_type: Any) -> core_schema.CoreSchema:
                 _validators.pattern_bytes_validator, serialization=ser, metadata=metadata
             )
         else:
-            raise PydanticSchemaGenerationError(f'Unable to generate pydantic-core schema for {pattern_type!r}.')
+            raise PydanticSchemaGenerationError(f"Unable to generate pydantic-core schema for '{pattern_type!r}'.")
```
Comment (samuelcolvin) [inline pydantic/_internal/_generate_schema.py:994]: this will result in two sets of quotes since you have `!r`.

**Review by samuelcolvin — COMMENTED** [review]
otherwise LGTM, please update.

**Comment by lig** [comment]
please review

**Review by dmontagu — APPROVED** [review]

**Pull request merged**